docs/syntaxStandard.md

```diff
@@ -0,0 +1,92 @@
+# MEML Syntax Standard
+
+This provides implementation details for the meml syntax. For an introduction to creating a language please see [crafting interpreters](https://craftinginterpreters.com/). A large part of this language standard guide is based off this book.
+
+## Language grammar
+
+Again, the language grammer here is based off of that for lox. Here page is a program. It contains everything in a file.
+
+```ts
+// This is the entry point into the program (a single file).
+//
+// DISCUSSION: How should scripting work? Should it be JS
+//             passthrough, should it be limited to inside
+//             custom tags, should it be LISP or should it
+//             be something more c-like (or python-like)
+page        → statement* EOF;
```

Comment:
Before I implement scripting, there needs to be a wider discussion that happens in relation to how scripting will work. I personally think that the combination of some JS (or lox) style language and meml page structure will work the best:
```js
let a = 5
let b = 6

(body
   (h1 "5 + 6 = " a + b)
)
```

However, scripting could also be made self-contained inside of tags:
```lisp
(body
    (h1 id="someID" "5 + 6 =")
    (script "let a = 5; let b = 6;document.getElementById('someID').appendText(a + b)")
)
```

There is also the potential for some form of lisp-style scripting. However, my personal opinion is that lisp is better suited for data structures.